docker/ci/Dockerfile

```diff
@@ -59,9 +58,9 @@ USER ${USER}
 
 RUN mkdir -p "$(rbenv root)"/plugins && \
     git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build && \
-    rbenv install 3.1.0 && \
+    rbenv install 3.1.7 && \
     eval "$(rbenv init -)" && \
-    rbenv shell 3.1.0 && \
+    rbenv shell 3.1.7 && \
```

Comment:
It'd be nice if we could just `ADD .ruby-version` and use that here, but we build this with the wrong directory context. Oh well.